pages/shared/groupWorkInfoContainer.ejs

```diff
@@ -0,0 +1,57 @@
+<div class="container-fluid">
+  <div class="row">
+    <% if (permissions.student_authz_join) { %>
+    <div class="col-sm bg-light py-4 px-4 border">
+      <p>This is a group assessment. Use your join code to invite others to join the group.</p>
+      <span class="badge badge-secondary">Group Name</span>
```

Comment:
I don't really like using badges as headers here. Perhaps just simple bold text with a colon?

**Group name:** myname
**Join code:** 32533

Reply:
Changed it. 
I think David will polish this UI design better with another PR very soon.